app/test_main.py

```diff
@@ -1,3 +1,43 @@
+import pytest
```

Comment:
Add a blank line between different groups of imports according to PEP-8

Reply:
Done